### ISSUE TITLE
Getting a list of all topics

### ISSUE THREAD

**Issue by rokstefanic** [mcollina/mosca#360]
Is it possible to get a list of all topics that (all) subscribers have subscribed to?

**Comment by mcollina** [comment]
This is a nice feature. I think @behrad would like to work on something similar, see #358.

**Comment by behrad** [comment]
no @mcollina , #358 is related to stored offline messages for a client to a topic. Something similar but more detailed.

**Comment by mcollina** [comment]
I'm just saying that there is a whole bunch of introspection needed on the persistence side.

**Comment by mesquka** [comment]
I can start implementing this feature on top of a Redis backend. Will need to look into compatibility with other backends.

**Comment by jsdario** [comment]
Hi, has there been any progress on this topic so far?

**Comment by mcollina** [comment]
It is not planned on my side.

**Comment by sprijk** [comment]
Would love to see this feature, to delete a topic and its nested topics recursively ;)

**Comment by jsdario** [comment]
I normally subscribe to the global topic `mqtt.subscribe('#')` and then keep a reference of them to list.
If done this on mosca broker, it could be easy to implement a stadistics channel to retrieve that list: `$stats/channels` e.g.

**Comment by zoyaali1** [comment]
Yes i can provide you real list that provide you all topics www.listfunda.com you can check list of all topics
